src/components/charts/detailed-bar.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import {generateColors} from './chart-builder'
+import {Bar} from 'react-chartjs-2'
+export default (title, chartRecipe) => {
+  const labels = chartRecipe.labels
+  const datasets = []
+
+  let index = 0
+  Object.keys(chartRecipe).forEach((key) => {
+    if (key !== 'type' && key !== 'labels') {
+      index += 1
+      datasets.push({
+        label: key,
+        borderWidth: 1,
+        data: chartRecipe[key],
+        backgroundColor: generateColors(index)[index - 1]
+      })
+    }
+  })
+  return <Bar
+    data={{
+      labels: labels,
+      datasets: datasets
+    }}
+    width='100%'
+    height='30vh'
+
+    options={{
+      maintainAspectRatio: true,
+      scales: {
+        yAxes: [{
+          ticks: {
+            min: 0
```

Comment:
Add stepSize = 1

Reply:
That looks absolutely ugly when a chart reaches a value of like 30...

Reply:
StepSize sets the minimum tho, not the max stepSize

Reply:
See http://www.chartjs.org/docs/latest/axes/cartesian/linear.html#step-size

Reply:
That is the documentation for suggestedMin and Max tho, the step size is constant...

Reply:
Ok you're right